backend/pkg/database/application.go

```diff
@@ -121,6 +122,8 @@ func (a *ApplicationTableHandler) Create(app *Application, params *TransactionPa
 		return fmt.Errorf("failed to create application: %v", err)
 	}
 
+	refreshMaterializedViewsIfNeeded(a.db, a.driverType)
+
```

Comment:
it also seems like this might need some sort of mapping to know which MaterializedViews should be updated based on what has changed. If we expand the views which are materialized in the future it would be weird/inefficent to refresh the CIS benchmark view when the packages DB had new data.

Reply:
updated